content/sensu-core/2.0/guides/install-check-executables-with-assets.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "How to install Check Executables using Assets"
+linkTitle: "Using Assets in Checks"
+weight: 100
+version: "2.0"
+product: "Sensu Core"
+platformContent: False
+menu: 
+  sensu-core-2.0:
+    parent: guides
+---
+
+## What are assets?
+Sensu assets are resources that checks can specify as dependencies. When an
+agent runs a check, it will ensure that all of the check's required assets
+are available to the agent during check runtime. If they aren't, the agent will
+install them by consulting each of the assets' URLs.
+
+## Why use assets?
+When configuration management is unavailable, assets can help manage runtime 
+dependencies such as scripts (e.g. check-haproxy.sh) and tar files (e.g. sensu-ruby-runtime.tar.gz)
+entirely within sensu. 
+
+## How to create a check that depends on an asset 
+
+### Creating an asset
+In this example, we'll create an asset from a `tar` archive, and show you how to
+create a new check with that asset as a dependency, as well as apply it to an
+existing check. To create an asset, we'll need a name, a URL to the asset location, 
+and a `SHA-512 checksum`.
+{{< highlight shell >}}
+$ sensuctl asset create check_website.tar.gz \
+  -u http://example.com/check_website.tar.gz \
+  --sha512 "$(sha512sum check_website.tar.gz | cut -f1 -d ' ')"
+{{< /highlight >}}
+
+If you're using a mac, you'll need to use `$(shasum check_website.tar.gz | cut -f1 -d ' ') to generate a checksum.
```

Comment:
Actually it should be `shasum -a 512` otherwise it uses the default algorithm, which is SHA-1 I believe.

Reply:
oh you are correct, I knew this and I still missed it lol :) fixing now!